src/main/java/seedu/duke/helper/Parser.java

```diff
@@ -40,6 +40,18 @@
         }
     }
 
+    public static ArrayList<String> parseViewDoctor(String description) {
+        //no specific doctor
+        if (description == null) {
+            return null;
+        }
+
+        //if specific doctor
+        ArrayList<String> parameters = new ArrayList<String>();
+        parameters.add(description);
+        return parameters;
```

Comment:
Just wondering, why did you return an array of strings instead of just returning back the String?